package.json

```
@@ -29,18 +29,18 @@
     "check-spelling": "cspell '**/*.{jsx,js,md,mdx}'"
   },
   "devDependencies": {
-    "@docusaurus/core": "3.7.0",
-    "@docusaurus/faster": "3.7.0",
-    "@docusaurus/preset-classic": "3.7.0",
-    "@docusaurus/theme-classic": "3.7.0",
-    "@docusaurus/theme-mermaid": "3.7.0",
-    "@docusaurus/types": "3.7.0",
+    "@docusaurus/core": "^3.8.1",
+    "@docusaurus/faster": "^3.8.1",
+    "@docusaurus/preset-classic": "^3.8.1",
+    "@docusaurus/theme-classic": "^3.8.1",
+    "@docusaurus/theme-mermaid": "^3.8.1",
+    "@docusaurus/types": "^3.8.1",
```

issue: We should pin version numbers otherwise it will be automatically bumped to next minor. We might also use `~` for auto patch updates

```suggestion
    "@docusaurus/core": "3.8.1",
    "@docusaurus/faster": "3.8.1",
    "@docusaurus/preset-classic": "3.8.1",
    "@docusaurus/theme-classic": "3.8.1",
    "@docusaurus/theme-mermaid": "3.8.1",
    "@docusaurus/types": "3.8.1",
```

---

Their automatic script seems to not be respecting version pinning :< Thanks for the catch. Change pushed 🙌